src/test/constructive_test.cpp

```diff
@@ -0,0 +1,123 @@
+#include "constructive_test.hpp"
+#include <iostream>
+#include <vector>
+#include <list>
+#include <unordered_map>
+#include <set>
+#include <random>
+#include <assert.h>
+
+namespace Test {
+
+enum Simbol {
+	SIGMA,
+	DECLARATION,
+	INVOKE,
+	VAR_NAME,
+	NUMBER,
+	BODY,
+	fn,
+	assign,
+	invoke,
+	paren,
+	end,
+	letterA,
+	letterB,
+	number1,
+	number2,
+	open_brackets,
+	close_brackets,
+	return_true
+};
+
+const std::set<Simbol> terminalSimbols {
```

Comment:
we prefer east-const

https://hackingcpp.com/cpp/design/east_vs_west_const.html